tripy/tripy/frontend/tensor.py

```diff
@@ -171,6 +172,9 @@ def dtype(self):
     def rank(self):
         return self.trace_tensor.rank
 
+    def stride(self):
```

Comment:
I am not sure why we need to expose stride method for frontend tensors. This information is mostly incorrect since after compilation the strides chosen by TRT for internal tensor can be changed.

Reply:
Agreed. I have removed the stride field.